src/main/scala/zio/prelude/package.scala

```diff
@@ -112,4 +114,21 @@ package object prelude
     def lessOrEqual(that: A)(implicit ord: Ord[A]): TestResult =
       assert(self)(isLessThanEqualTo(that))
   }
+
+  implicit class AnySyntax[A](private val a: A) extends AnyVal {
+
+    @silent("side-effecting nullary methods are discouraged")
+    /* Ignores the value, if you explicitly want to do so and avoids "Unused value" compiler warnings. */
+    def ignore: Unit = ()
+
+    /** Applies function `f` to a value `a`, like `f(a)`, but in flipped order and doesn't need parentheses. Can be chained, like `x |> f |> g`. */
+    def |>[B](f: A => B): B = f(a)
```

Comment:
Does this conflict with the new `|>` operator in Scala 2.13?

Reply:
AFAIK there is no `|>` operator in Scala 2.13

Reply:
I think these both got added here: https://www.scala-lang.org/api/current/scala/util/ChainingOps.html.

Reply:
Ah, I forgot about `tap` being there. Yes, `pipe` got added, but
 * it's only in 2.13
 * needs an explicit import
 * it's not `|>` which is an "operator"
 * `anArgument |> someMethod |> someOtherFunction` is much easier to read than `anArgument.pipe(someMethod).pipe(someOtherFunction)`, especially if the names are more involved

Reply:
I hear you. I'm just a little worried we're adding generally nice syntax but not syntax that is specifically related to this library. Like I like both the `|>` and `tap` methods but I'm not sure I would expect them to automatically get added to any type by using ZIO Prelude.

Reply:
Noted. I can split it into a new PR and we can move forward with `>>>` . In the meantime, we can think of +/- of `|>` & co.

Reply:
Sounds good. You want to just include the `AssociativeComposeSyntax` and we can open an issue for the `AnySyntax`?

Reply:
OK, removed
new PR: https://github.com/zio/zio-prelude/pull/227